trantor/net/ssl/SSLConnectionSkipped.cc

```diff
@@ -0,0 +1,80 @@
+/**
+ *
+ *  SSLConnectionSkipped.cc
+ *  An Tao
+ *
+ *  Public header file in trantor lib.
+ *
+ *  Copyright 2018, An Tao.  All rights reserved.
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the License file.
+ *
+ *
+ */
+
+#include <trantor/net/ssl/SSLConnection.h>
+#include <trantor/utils/Logger.h>
+#include <trantor/net/inner/Channel.h>
+#include <trantor/net/inner/Socket.h>
+#include <thread>
+
+using namespace trantor;
+
+namespace trantor
+{
+class SSLContext
+{
+};
+class SSLConn
+{
+};
+
+void initOpenSSL()
+{
+    LOG_FATAL << "SSL is not supported!";
```

Comment:
Hmm, I'd probably word these 3 log strings as something like `"SSL is not supported due to not compiling with OpenSSL"` or so, to give a hint of what they need to do to fix it.